lib/spawn-runner.js

```diff
@@ -49,10 +49,7 @@ const readFileAsync = Promise.promisify(readFile);
 const unlinkAsync = Promise.promisify(unlink);
 
 export function convertPathToPattern(path :string) :string {
-  if (process.platform.match(/^win/)) {
```

Comment:
Maybe normalize path will be better solution for this issue?
https://nodejs.org/dist/latest-v7.x/docs/api/path.html#path_path_normalize_path

Which windows path does not work? `const path = 'C:\\path\\to\\spec\\some.spec.js';`?